test/bin/get_rel_version_repo.sh

```diff
@@ -58,30 +60,29 @@ get_current_release_from_sub_repos() {
 	echo ""
 }
 
-# get_crel_version_repo attempts to obtain full version string and repository url
-# for "already released RPMs for current release" - this includes ECs and RCs from
+# get_version_repo attempts to obtain full version string and repository url
+# for "already released RPMs for minor release" - this includes ECs and RCs from
 # http mirrors, and packages in the ocp repository.
 #
 # Failed attempt to obtain version/repo is not a failure, at the start of each release,
 # there might not be an EC yet. In such case variables exported below might be empty,
 # and such osbuild source, blueprint, and test scenarios should be skipped.
-get_crel_version_repo() {
+get_rel_version_repo() {
 	local -r minor="${1}"
 
 	CURRENT_RELEASE_VERSION=$(get_current_release_from_sub_repos "${minor}")
```

Comment:
If this script is sourced and used for different settings, we should probably make all of the variables in these functions locals to avoid stomping over the values in the script that imports this code.